rgd-client/rgd_client/utils.py

```diff
@@ -49,7 +49,7 @@ def limit_offset_pager(session: Session, url: str, **kwargs) -> Generator[Dict,
         r.raise_for_status()
 
         # Yield results
-        results = r.json()['results']
+        results = r.json()
```

Comment:
@AlmightyYakob, I've had to change this back and forth a few times. How should this be?

Reply:
It should be `results = r.json()`, otherwise you're just yielding a dict, and need to manually extract the `results` field. @venkatabhishek What was the reason for this change?

Reply:
When I print the contents of `r.json()` when running something like `client.search(query=json.dumps(bbox), predicate='intersects')` as in the README example, it is just a list of the returned spatial entries. With `r.json()['results']` it raises an error related to this (i.e. TypeError: list indices must be integers or slices, not str)

Reply:
The search endpoint returns a paginated response, not a list directly, correct @banesullivan? What you're observing @venkatabhishek implies that the search endpoint is returning a list directly.

Reply:
Hmm.. I'm not exactly sure what is happening here. The search endpoints use `rest_framework.generics.ListAPIView` which I am observing returning a list directly. It seems to me that using `limit` in the Python client to paginate the search results isn't currently doing anything?

Reply:
Okay my bad, it seems the production is not up to date with that endpoint change.

The `limit_offset_pager` function is intended to be used on endpoints which are making use of limit offset paging. If the search endpoint isn't intended to paginate the results, then we shouldn't use this function, and should just call that endpoint directly in `Rgdc.search`.

I think the easiest thing to do right now is to change this line back to `results = r.json()['results']`, and remove the call to `limit_offset_pager` in `Rgdc.search` (and anywhere else it may be necessary). If pagination is meant to be applied to that endpoint and fixed later on, then these tests should fail (:tada: yay tests) and we can add the use of `limit_offset_pager` back in. Regardless, the api surface for this method shouldn't change (it still returns a list in either case).

Does this sound good to both of you @banesullivan @venkatabhishek?

Reply:
Addressed in bdb6b8c so I'm happy